### DISCUSSION TITLE
Can Frigate work on dashcam footage?

### DISCUSSION THREAD

**Discussion by anthonyma94** [blakeblackshear/frigate#3217] [General]
I'm looking for a self hosted video AI and came across this. My usage is simple: when my car gets home, Home Assistant runs an action to download all the footage from my dashcam onto my server. Can Frigate then run on the videos and look for, say car crashes (I saw something about using custom models as well?), and sort them to a different folder?

**Comment by NickM-27** [comment]
This technically could work but there are a number of "technical difficulties" that would need to be ironed out. 

[As shown in the docs](https://docs.frigate.video/contributing#setup) frigate can work with an mp4 file, but there would need to be a lot of outside automation / scripting to run the container and then stop it to adjust the config to select a new mp4 file.

Also, Frigate does work on tensorflow lite models but I have not heard of any models that specifically detect car crashes and using the general car tracking seems like it would be quite difficult to ascertain when a crash had occurred.

**Reply by anthonyma94** [reply]
I see, I'll have to look for some other solution. Thank you!

**Comment by mballoni** [comment]
Hi!  Have you found a solution for this ?

I'm looking into a similar scenario but I would not need crash detection. As long as it automatically keeps the footage for a certain amount of time I'd be happy (initially).
